apps/web/src/components/happening-sidebar.tsx

```diff
@@ -463,6 +463,14 @@ export const HappeningSidebar = async ({ event }: EventSidebarProps) => {
                 <Countdown toDate={userRegistrationStart} />
               </SidebarItem>
             )}
+
+            {Boolean(event.externalLink) && (
```

Comment:
TypeScript er skækk. `Boolean` gjør faktisk at TS ikker får til å infere typen riktig.

```suggestion
            {event.externalLink && (
```